src/Microsoft.TestPlatform.CrossPlatEngine/TestEngine.cs

```diff
@@ -109,29 +112,13 @@ public ITestExtensionManager GetExtensionManager()
             return this.testExtensionManager ?? (this.testExtensionManager = new TestExtensionManager());
         }
 
-        /// <summary>
-        /// Retrieves the default test host manager for this engine.
-        /// </summary>
-        /// <param name="runConfiguration">
-        /// The run Configuration.
-        /// </param>
-        /// <returns>
-        /// An instance of the test host manager.
-        /// </returns>
-        public ITestRuntimeProvider GetDefaultTestHostManager(RunConfiguration runConfiguration)
+        /// <inheritdoc/>
+        public ITestRuntimeProvider GetDefaultTestHostManager(string runSettingsXml)
```

Comment:
May be this need not be an interface method? It's not getting the `Default` test host manager, it just gets the runtime provider which can understand a runsettings.